problems/coj_problems.go

```diff
@@ -29,91 +54,86 @@ func generate_request_url(method, url_requested string, parameters map[string]st
     return request.URL.String(), nil
 }
 
-// COJJudgement is the struct to represent a COJ judment for a submition.
-type COJJudgement struct {
-    Id int
-    Date string
-    User string
-    Prob int
-    Judgment string
-    Errortestcase int
-    Time int
-    Memory string
-    Tam string
-    Lang string
-}
-
-// COJJudgmentResponse is the struct to represent a collection of COJ Judgments.
-type COJJudgmentResponse struct {
-    Collection [] COJJudgement
-}
-
+// createNetClient generates a http client to make request.
+// 
+// It returns a pointer of type http.Client to make request to servers.
+// We do the request in this way so we can control the timeout, otherwise we may have troubles
+// if the server we are requesting for fails.
+func createNetClient() (*http.Client){
+    const timeoutRequest = 10 // Requests timeouts
```

Comment:
You could do something like `const timeoutRequest = 10 * time.Second` right here.